R/Neural_Network/program/Neural_Network_ETH.py

```diff
@@ -10,8 +10,6 @@
 import os
 import numpy as np
 import pickle
-import collections
-from sklearn.feature_selection import RFE
```

Comment:
See above

Reply:
I'd say this is worth keeping given the last update was ~5 years ago.